devstudio/installer/pom.xml

```diff
@@ -0,0 +1,83 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>com.jboss.jbds.integration-stack</groupId>
+		<artifactId>parent</artifactId>
+		<version>8.0.1-SNAPSHOT</version>
+	</parent>
+	<artifactId>installer</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>pom</packaging>
+	<name>IBDSIS Installer</name>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-dependency-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>copy-p2-director</id>
+						<phase>prepare-package</phase>
+						<goals>
+							<goal>unpack</goal>
+						</goals>
+						<configuration>
+							<artifactItems>
+								<artifactItem>
+									<groupId>com.jboss.jbds.integration-stack</groupId>
+									<artifactId>site</artifactId>
+									<version>8.0.1-SNAPSHOT</version>
+									<type>zip</type>
+									<outputDirectory>target/integration-stack</outputDirectory>
+								</artifactItem>
+							</artifactItems>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+
+			<plugin>
+				<groupId>com.googlecode.maven-download-plugin</groupId>
+				<artifactId>download-maven-plugin</artifactId>
+				<version>1.2.1</version>
+				<executions>
+					<execution>
+						<id>fetch-target-platform</id>
+						<phase>generate-sources</phase>
+						<goals>
+							<goal>wget</goal>
+						</goals>
+						<configuration>
+							<url>file:///home/eskimo/data/Projects/jbdevstudio/4.3.x/fork/jbds-releng/jbdevstudio-product/installer/target/jboss-devstudio-9.0.0.Alpha2-v20150326-1946-installer-standalone.jar</url>
```

Comment:
so we need to fine a stable url for these somewhere. I guess our internal release archive could work.